docs/software/sciapps/quantumespresso.md

```diff
@@ -1,4 +1,66 @@
 [](){#ref-uenv-quantumespresso}
 # Quantum ESPRESSO
-!!! todo
-    complete docs
+
+Quantum ESPRESSO is an integrated suite of Open-Source computer codes for electronic-structure calculations and materials modeling at the nanoscale. It is based on density-functional theory, plane waves, and pseudopotentials:
+
+PWscf (Plane-Wave Self-Consistent Field)
+FPMD (First Principles Molecular Dynamics)
+CP (Car-Parrinello)
+
+## ALPS (GH200)
+### Setup
+Download the uenv image for QuantumESPRESSO:
+
+
+### List available images
+
+```bash
+uenv image find quantumespresso
+```
+
+### Pull the image of interest
+
+```
+uenv image pull quantumespresso/v7.4:v2
+```
+
+
+QuantumESPRESSO can be compiled from source using the above uenv. The procedure is described in https://eth-cscs.github.io/alps-uenv/uenv-qe/#building-a-custom-version.
+
+### How to run
+
+The following sbatch script can be used as a template.
```

Comment:
Might be useful to link to https://eth-cscs.github.io/cscs-docs/running/slurm/#one-rank-per-gpu? I think a full example here is good, but I think it's also good to link to the more generic documentation should the user need/want to modify things.

Reply:
I've added the link to one rank and also for multiple ranks per gpu.